multiversxetl/jobs/transform_job.py

```diff
@@ -53,6 +53,12 @@ def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
 class BlocksTransformer(Transformer):
     def _do_transform(self, data: Dict[str, Any]) -> Dict[str, Any]:
         data.pop("pubKeyBitmap", None)
+
+        # Remove "epochStartShardsData.pendingMiniBlockHeaders.reserved".
```

Comment:
What is the purpose of the `reserved` field?

Reply:
Is a field from the headers that are used in the protocol, just exposed it also in the ES database.